package.json

```diff
@@ -25,8 +26,9 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "bower": "^1.8.0",
+    "cross-env": "^5.1.3",
     "eslint": "^3.17.0",
```

Comment:
Feel free to update to latest eslint and polymer-cli... this is what we've been using across the board for the shared components... although i would imagine there are even newer versions released in the last week.  ;)

```
"eslint": "^4.15.0",
"eslint-config-brightspace": "^0.4.0",
"eslint-plugin-html": "^4.0.1",
"polymer-cli": "^1.5.7"
```